app/models/Database.scala

```diff
@@ -0,0 +1,149 @@
+    moviesList(2),
+    ScreeningRoom(1))
+  screenigsList += Screening(11,
+    new DateTime().withYear(2023).withMonthOfYear(1).withDayOfMonth(15).withHourOfDay(20).withMinuteOfHour(30),
+    new DateTime().withYear(2023).withMonthOfYear(1).withDayOfMonth(15).withHourOfDay(23).withMinuteOfHour(40),
+    moviesList(0),
+    ScreeningRoom(3))
+  screenigsList += Screening(1,
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(20).withHourOfDay(17).withMinuteOfHour(30),
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(20).withHourOfDay(19).withMinuteOfHour(30),
+    moviesList(1),
+    ScreeningRoom(1))
+  screenigsList += Screening(2,
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(22).withHourOfDay(17).withMinuteOfHour(40),
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(22).withHourOfDay(19).withMinuteOfHour(40),
+    moviesList(2),
+    ScreeningRoom(2))
+  screenigsList += Screening(3,
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(23).withHourOfDay(18).withMinuteOfHour(15),
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(23).withHourOfDay(20).withMinuteOfHour(15),
+    moviesList(3),
+    ScreeningRoom(3))
+  screenigsList += Screening(4,
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(24).withHourOfDay(20).withMinuteOfHour(35),
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(24).withHourOfDay(22).withMinuteOfHour(35),
+    moviesList(4),
+    ScreeningRoom(1))
+  screenigsList += Screening(5,
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(25).withHourOfDay(21).withMinuteOfHour(30),
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(25).withHourOfDay(23).withMinuteOfHour(30),
+    moviesList(5),
+    ScreeningRoom(3))
+  screenigsList += Screening(6,
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(26).withHourOfDay(21).withMinuteOfHour(0),
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(26).withHourOfDay(22).withMinuteOfHour(0),
+    moviesList(6),
+    ScreeningRoom(2))
+  screenigsList += Screening(8,
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(27).withHourOfDay(17).withMinuteOfHour(0),
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(27).withHourOfDay(19).withMinuteOfHour(0),
+    moviesList(1),
+    ScreeningRoom(1))
+  screenigsList += Screening(9,
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(28).withHourOfDay(17).withMinuteOfHour(0),
+    new DateTime().withYear(2022).withMonthOfYear(12).withDayOfMonth(28).withHourOfDay(19).withMinuteOfHour(0),
+    moviesList(2),
+    ScreeningRoom(2))
+
+  val reservationsList = new mutable.ListBuffer[Reservation]()
+
+
+  def validateReservation(reservation: Reservation): Boolean = {
+
+
+    var idCorrectness = false
+    for(screening <- screenigsList){
```

Comment:
Czy jest jakaś metoda, której możemy użyć na liście by sprawdzić czy występuje w niej element spełniający pewne warunku? Tak by nie trzeba było za każdym razem przechodzić przez n elementów?

Reply:
Można by zrobić to w ten sposób
val idCorrectness = screenigsList.exists(_.id == reservation.screeningId)